src/App.tsx

```diff
@@ -8,6 +8,7 @@ import { PublicRiddlesPage } from './pages/PublicRiddlesPage';
 import { ReceivedRiddlesPage } from './pages/ReceivedRiddlesPage';
 import { MyRiddlesPage } from './pages/MyRiddlesPage';
 import { theme } from './theme';
+import { CreateRiddlePage } from './pages/CreateRiddlePage.tsx';
```

Comment:
Zase mi řekneš, že tam je v importech spoustu přípon navíc 😄 - zkontroluju pak na závěr, teď je potřeba jen dořešit pár nejasností viz dále.